### PR TITLE
deprecate `setup ui windicss` (project is sunsetting) 

### PR DESCRIPTION
**Pull request by thedavidprice**
_PR 1 of 2_

The WindiCSS project is sunsetting. One of the recommended replacements is Tailwind CSS, which is already supported. See: https://windicss.org/posts/sunsetting.html

This PR indicates `deprecated` in the setup command and CLI docs. (target v5.4)

The follow-up PR will remove the setup command entirely. (target v6) https://github.com/redwoodjs/redwood/pull/8598

Note: Projects using Windi CSS will not be affected when upgrading. Redwood v6 switches to Vite as the default bundler (Webpack supported but deprecated). At that time, if projects using Windi CSS upgrade to v6 and want to use Vite, they'll need to follow the Windi + Vite installation guide to manually make the changes to upgrade: https://windicss.org/integrations/vite.html

### CODE REVIEW

File: packages/cli/src/commands/setup/ui/libraries/windicss.js [hunk 1]
```diff
@@ -8,10 +8,24 @@ import { errorTelemetry } from '@redwoodjs/telemetry'
 
 import { getPaths } from '../../../../lib'
 import c from '../../../../lib/colors'
+import { printSetupNotes } from '../helpers'
```
Comment (jtoar) [inline packages/cli/src/commands/setup/ui/libraries/windicss.js:11]: Just tried running this locally and it seems like this file doesn't exist at this location; this specific helper is co-located with the deploy commands at the moment. Don't see why we can't move it, we probably just never had a reason to

File: packages/cli/src/commands/setup/ui/libraries/windicss.js [hunk 2]
```diff
@@ -8,10 +8,24 @@ import { errorTelemetry } from '@redwoodjs/telemetry'
...
+  'Set up WindiCSS\n' +
+  'For more information:\n' +
+  'https://windicss.org/posts/sunsetting.html'
+
+export const notes = [
+  c.error('DEPRECATED option not officially supported'),
+  '',
+  'Windi CSS is sunsetting. For more information:',
+  'https://windicss.org/posts/sunsetting.html',
+  '',
```
Comment (jtoar) [inline packages/cli/src/commands/setup/ui/libraries/windicss.js:26]: I'm seeing more empty space at the bottom than the top:

```
  ┌────────────────────────────────────────────────┐
  │                                                │
  │ DEPRECATED option not officially supported     │
  │                                                │
  │ Windi CSS is sunsetting. For more information: │
  │ https://windicss.org/posts/sunsetting.html     │
  │                                                │
  │                                                │
  └────────────────────────────────────────────────┘
```

```suggestion
```

File: packages/cli/src/commands/setup/ui/libraries/windicss.js (hunk 1 above)
Comment (jtoar) [reply packages/cli/src/commands/setup/ui/libraries/windicss.js]: Fixed in https://github.com/redwoodjs/redwood/pull/8597/commits/d80b8dfd8240de56eee584c4172e6a8b79e02693

**Comment by jtoar** [comment]
I'm seeing errors in the CLI tests I've seen in my other PR here: https://github.com/redwoodjs/redwood/pull/8448. I'm not savvy enough with Jest to know what I'm doing wrong with the mocks. And it's late so I'll try again another time

**Comment by thedavidprice** [comment]
Thanks for the help with this one @jtoar Setting to auto-merge now.

**Comment by Olyno** [comment]
Why not migrate to Unocss?

**Pull request merged**